.travis.yml

```diff
@@ -4,7 +4,7 @@ script:
 - |
   stat=0
   for projfile in $(ls configs/projects/[a-z]*.rb); do
-    for plat in el-7-x86_64 ubuntu-18.04-amd64 osx-10.13-x86_64 windows-2012r2-x64; do
+    for plat in el-7-x86_64 ubuntu-18.04-amd64 ubuntu-18.10-amd64 osx-10.13-x86_64 windows-2012r2-x64; do
```

Comment:
Do we need to add it here? We already have a representative Ubuntu platform. Won't block the PR on this, just curious is all.

Reply:
Yeah, no need to add the platform to travis.